### DISCUSSION TITLE
Training using tensorflow

### DISCUSSION THREAD

**Discussion by akniloy6** [jr-robotics/robo-gym#40] [General]
Hello. Is there Any examples available using robo-gym environment to train a model developed in tensorflow/keras? Would be much appreciated as it would save me some time

**Comment by friedemannzindler** [comment]
Hi @AKNiloy. 

Unfortunately, we do not have any examples right now, but in principle the algorithms provided by stable-baselines should work with robo-gym.

stable-baselines: https://github.com/hill-a/stable-baselines
short-example: https://github.com/jr-robotics/robo-gym/blob/master/docs/examples/stable-baselines/td3_script.py

There are also some code samples from an old issue: https://github.com/jr-robotics/robo-gym/issues/33


We are currently working with pytorch and also thinking about releasing this, but not in the near future. 

Hope this helps you a little.
Friedemann

**Reply by akniloy6** [reply]
I found a working example with robo-gym using TF. will add the repo soon. Thanks anyways. Looking forward to the pytorch implementation!!

**Reply by coral26** [reply]
@AKNiloy  ,  Would you please share the working example with robo-gym using TF that you found? Does it work with TF-Agents?

**Reply by akniloy6** [reply]
I'm kinda busy atm I will share by next week hopefully. Though so far it only worked with the non-obstacle version of the Mir robot env. I didnt work with TF agents so I dont know but I think it should work.
